README.md

```diff
@@ -210,6 +210,137 @@ This app uses [RSpec](http://rspec.info/) for Rails tests.
 
     rake parallel:spec
 
+## Environment variables used by the application
 
+### Sending emails
 
+#### To prisons
 
+We use MessageLabs to send emails to prisons because they're on the
+Government Secure Intranet (GSI). On preprod, we use Sendgrid to send
+to maildrop so we can get access to them easily. You will need to
+configure them if you want to send email when submitting a booking
+request locally.
+
+Source: `app/mailers/prison_mailer.rb`
+
+- `GSI_SMTP_HOSTNAME`
+- `GSI_SMTP_PORT`
+- `GSI_SMTP_DOMAIN`
+
+#### To visitors
+
+These SMTP settings are used to send emails to visitors. Because these using SendGrid's SMTP server, they must be sent via SendGrid. You will need to configured them if you want to try and test email by submiting a booking request locally.
+
+Source: `config/environments/production.rb`
+
+
+- `SMTP_HOSTNAME`
+- `SMTP_PORT`
+- `SMTP_DOMAIN`
+- `SMTP_USERNAME`
+- `SMTP_PASSWORD`
+
+### Change default value of
+
+#### The Service Url
+
+This URL is used to configure the session store host and to generate links in emails. For development and test environment is set to `localhost`.
+
+Source: `config/environments/production.rb` and `config/initializers/session_store.rb`
+
+- `SERVICE_URL`
+
+#### The Prison Data File
+
+The default prison data file is `prison_data_staging.yml`, but you can configure a different one.
+
+Source: `config/application.rb`
+
+- `PRISON_DATA_FILE`
+
+#### The Message Encryptor Secret Key
+
+The key is a base 64-encoded value and is used to encrypt serialised data that are transmitted in URLs in the emails that are sent to prison staff (to process requests) and visitors (to view or cancel requests).
+
+Source: `config/initializers/encryptor.rb`
+
+- `MESSAGE_ENCRYPTOR_SECRET_KEY`
+
+#### The GovUk Start Page
+
+On production, we set this to [https://www.gov.uk/prison-visits](https://www.gov.uk/prison-visits) because that's the official start page for the service, but you can configure a different one.
+
+Source: `config/routes.rb`
+
+- `GOVUK_START_PAGE`
+
+### Setting up
+
+#### The Google Analytics ID.
+
+Source: `config/application.rb`
+
+- `GA_TRACKING_ID`
+
+#### The Prison Estate IP
+
+Users on these IP addresses can access metrics without needing to log in, the prison booking admin pages and the prison staff info pages which is a comma-separated list.
```

Comment:
This might be better as:

> This is a comma-separated list of IP addresses. Users on these addresses can access metrics, the prison booking admin pages, and the prison staff info pages without needing to log in.